### ISSUE TITLE
Add line boundary for RIGR

### ISSUE THREAD

**Issue by mtaylorlong** [nationalparkservice/places-data#656]
Rio Grande WSR is an odd boundary that needs fixing in PT3. It's a very thin boundary, so it would be best symbolized as a line until zoom 14. Also, it's label is completely missing, so we'll have to figure that out.

cc: @cwlawlis802

**Comment by chadlawlis** [comment]
It looks like the unit boundary may be cutting off at two different tile edges? cc @mtaylorlong 

Western edge @ http://insidemaps.nps.gov/park-tiles-3/#19/29.73504/-102.65588.

Eastern edge @ http://insidemaps.nps.gov/park-tiles-3/#19/29.79664/-101.95284.

The boundary geometry looks to be correct:

Map on NPS.gov:
![rigr_nps_gov](https://cloud.githubusercontent.com/assets/1238636/16051608/ab7e7b1e-321d-11e6-8551-02c3a1ae6404.png)

Geometries (RIGR + BIBE + AMIS) in Places:
![rigr_bibe_amis](https://cloud.githubusercontent.com/assets/1238636/16051648/d6dd9902-321d-11e6-86e4-4497ec8451d0.png)

There is a label but it is all the way at the southeastern USA/Mexico border: http://insidemaps.nps.gov/park-tiles-3/#14/25.9545/-97.1436. So, this will need to be moved.

**Comment by chadlawlis** [comment]
Label move done via:

``` sql
BEGIN;
UPDATE parks_label
SET geom_label = (SELECT ST_GeomFromText('POINT(-102.31876373291016 29.88095077689225)', 4326))
WHERE label_id = 141;

UPDATE parks_label
SET geom_label = ST_Transform(ST_SetSRID(geom_label, 4326), 3857)
WHERE label_id = 141;
COMMIT;
```

This will put the label in a similar position as on the print map from above. Will confirm once live.

**Comment by mtaylorlong** [comment]
@cwlawlis802: RIGR is a great candidate for becoming a line park, just like BUFF. Let's set `simp_type='line'` and add a line geometry.

This polygon was definitely made from a line. Any chance we have that line?

**Comment by chadlawlis** [comment]
Agreed - I've been searching for the line geometry, to no avail. Generating a centerline from a polygon in QGIS or PostGIS is more complicated than imagined as well. We'll get a hold of it one way or another...

**Comment by chadlawlis** [comment]
The label is now live.

The boundary geometry is actually looking better, as well - it does not seem to be getting cut off like it was before. It would still be best if we symbolized it as a line up until z14, but looking better in the mean time at least.
